integration/benchnet2/Makefile

```diff
@@ -1,11 +1,33 @@
+# default values that can callers override when calling target
+ACCESS_NODES=2
+COLLECTION_NODES=6
+CONSENSUS_NODES=3
+EXECUTION_NODES=2
+VERIFICATION_NODES=1
```

Comment:
You may consider using the same variable names as the localnet: https://github.com/onflow/flow-go/blob/3b26945ec29f299ed511bf12b4ef1d0ca3cc6a89/integration/localnet/Makefile#L1-L9